src/connectErrorEpic.js

```diff
@@ -3,6 +3,9 @@ const connectErrorEpic = action$ => action$
   .map((action) => {
     const { meta, payload: e } = action;
     const op = action.type === '@@stripes-connect/FETCH_ERROR' ? 'GET' : e.type;
+
```

Comment:
Is this part of the present PR's functionality, or an unrelated change?

Reply:
Ooohhh ... maybe this is the new stripes-connect functionality that we're depending on? Great if so, but the stripes-connect `CHANGELOG.md` says nothing about what's in v3.1.2. Maybe you could fix that?

Reply:
_sigh_ save yourself some time and ignore everything I just wrote. I realised my copy of stripes-connect was outdated. I `git pull`ed and the `CHANGELOG.md` explains perfectly.

(Only one nit on that score: the entry does not reference a Jira  for `throwErrors` -- was there not one?)

Reply:
Ahhh, it doesn't, good nit. :) It was STCON-67

Reply:
Anyway: go ahead and merge. Thanks for handling this.